api/types/types.proto

```diff
@@ -518,7 +518,8 @@ message ClusterConfigSpecV3 {
     string ClusterID = 2 [ (gogoproto.jsontag) = "cluster_id" ];
 
     // Audit is a section with audit config
-    AuditConfig Audit = 4 [ (gogoproto.nullable) = false, (gogoproto.jsontag) = "audit" ];
+    ClusterAuditConfigSpecV2 Audit = 4
+        [ (gogoproto.nullable) = true, (gogoproto.jsontag) = "audit,omitempty" ];
```

Comment:
hmm, what will happen when the auth server reads an existing `ClusterConfig` from the backend?
these two types are not compatible, will it fail to parse (or just zero all the fields)?
or does the auth server always upsert the `ClusterConfig` on startup and this will be overwritten?

Reply:
When unmarshalling from JSON, the two types should be viewed as compatible, I think...?

Anyway, as you alluded to, the case is that the `ClusterAuditConfig` is not read from the backend during auth server initialization -- it's always overwritten with what's specified in `teleport.yaml`.